src/expr/src/scalar/func.rs

```diff
@@ -4704,6 +4715,7 @@ pub enum VariadicFunc {
     RegexpMatch,
     HmacString,
     HmacBytes,
+    RepeatString,
```

Comment:
This seems like it wants to be a `BinaryFunc`, no?

Reply:
True! Morning brain, will move.